core/src/wallet/tezos/explorers/NodeTezosLikeBlockchainExplorer.cpp

```diff
@@ -210,32 +210,37 @@ namespace ledger {
             return "";
         }
 
+        Future<std::shared_ptr<BigInt>> NodeTezosLikeBlockchainExplorer::getHelper(const std::string &url,
+                                                                                   const std::string &field,
+                                                                                   const std::unordered_map<std::string, std::string> &params) {
+            bool parseNumbersAsString = true;
+            auto networkId = getNetworkParameters().Identifier;
+
+            std::string p;
+            for (auto &param : params) {
+                std::string separator = p.empty() ? "?" : "&";
```

Comment:
This test is going to be done for all elements while, after the first iteration, we know it will always be `true`.